File: Math/sol.py
# https://codeforces.com/problemset/problem/486/A

def sumOdds(n):
    if not (n & 1):
        return ((n // 2) * (n // 2)) * 2
    return (((n + 1) // 2) * ((n + 1) // 2)) * 2

def sol():
    n = int(input())
    print((n * (n + 1) // 2) - sumOdds(n))

t = 1
while t:
    sol()
    t -= 1

File: Math/sol.cpp
/*
https://codeforces.com/problemset/problem/1742/A
*/

#include <bits/stdc++.h>


typedef long long int ll;
typedef std::vector<int> vec;
using namespace std;

void sol(){
    int a, b, c;
    cin >> a >> b >> c;
    if(a + b == c || b + c == a || a + c == b){
        cout << "YES\n";
        return;
    }
    cout << "NO\n";
}

int main(){
    ios::sync_with_stdio(0);
    cin.tie(0);
              
    int t;
    cin >> t;
    while(t--){
        sol();
    }


    return 0;
}